src/sentry/apidocs/parameters.py

```diff
@@ -70,6 +70,26 @@ class GLOBAL_PARAMS:
         description="The name of environments to filter by.",
     )
 
+    @staticmethod
+    def NAME(description: str, required: bool = False) -> OpenApiParameter:
```

Comment:
since we're now adding methods to this class, I think we should redo the class casing to reflect this:

`GLOBAL_PARAMS` -> `GlobalParams`
keep the constants capitalized

and then the methods can be snake_case.

Reply:
I can do this in a followup PR, just checked and renaming the class would update ~20 files and clutter this PR. I'll include the methods snack_case change here tho

Reply:
gotcha, sounds good.